### PR TITLE
rename: use Kalix in docs

### PR DESCRIPTION
**Pull request by octonato**

### CODE REVIEW

**Comment by octonato** [comment]
> Fails because of missing docker images in `kalix-public`. Can we try to use the old one from `akkaserverless-public`?

ok, I will revert this one

**Comment by jrudolph** [comment]
> > Fails because of missing docker images in `kalix-public`. Can we try to use the old one from `akkaserverless-public`?
> 
> ok, I will revert this one

Now, `kalix-public` is available but I guess we still need to publish the docs infrastructure there, yet?

**Comment by octonato** [comment]
> > > Fails because of missing docker images in `kalix-public`. Can we try to use the old one from `akkaserverless-public`?
> > 
> > 
> > ok, I will revert this one
> 
> Now, `kalix-public` is available but I guess we still need to publish the docs infrastructure there, yet?

I don't know if I'm following you.

**Comment by jrudolph** [comment]
> I don't know if I'm following you.

Sorry, going back to use the old one should be fine for now :)

File: docs/src/modules/developing/pages/development-process-java.adoc [hunk 1]
```diff
@@ -30,7 +30,7 @@ TIP: Kickstart a project using the xref:java:quickstart-template.adoc[project te
 [#_implement_components]
 == Implement components
 
-Stateful services can implement https://developer.lightbend.com/docs/akka-serverless/reference/glossary.html#value_entity[Value Entity] or https://developer.lightbend.com/docs/akka-serverless/reference/glossary.html#event_sourced_entity[Event Sourced Entity] and https://developer.lightbend.com/docs/akka-serverless/reference/glossary.html#view[View] components. Stateless services implement https://developer.lightbend.com/docs/akka-serverless/reference/glossary.html#action[Actions]. Typically, a stateful service should only have one Entity type, but can also include Actions and Views.
+Stateful services can implement https://developer.lightbend.com/docs/kalix/reference/glossary.html#value_entity[Value Entity] or https://developer.lightbend.com/docs/kalix/reference/glossary.html#event_sourced_entity[Event Sourced Entity] and https://developer.lightbend.com/docs/kalix/reference/glossary.html#view[View] components. Stateless services implement https://developer.lightbend.com/docs/kalix/reference/glossary.html#action[Actions]. Typically, a stateful service should only have one Entity type, but can also include Actions and Views.
```
Comment (patriknw) [inline docs/src/modules/developing/pages/development-process-java.adoc:33]: `developer.lightbend.com/docs/kalix` should be changed to `docs.kalix.io`

File: docs/src/modules/java/pages/actions.adoc [hunk 2]
```diff
@@ -64,7 +64,7 @@ Scala::
 ----
 include::example$scala-fibonacci-action/src/main/scala/com/example/fibonacci/FibonacciActionGenerated.scala[tag=generated-action]
 ----
-<1> Extends the generated `AbstractFibonacciAction`, which extends link:{attachmentsdir}/scala-api/com/akkaserverless/scalasdk/action/Action.html[`Action` {tab-icon}, window="new"].
+<1> Extends the generated `AbstractFibonacciAction`, which extends link:{attachmentsdir}/scala-api/com/kalix/scalasdk/action/Action.html[`Action` {tab-icon}, window="new"].
```
Comment (patriknw) [inline docs/src/modules/java/pages/actions.adoc:67]: `com/kalix` should be `kalix` if it's a package name

**Comment by octonato** [comment]
`gcr.io/kalix-public/kalix-docbuilder` is making the build to fail, so I revert back to old image for now. 

I would like to get this one merged soon because we have other PRs also making textual changes and we may end up doing double work. 

We can fix the image later.

**Comment by octonato** [comment]
For the record, I'm preparing the PR to publish the new docbuilder image.

**Comment by octonato** [comment]
imagine will be published after we merge this other one: https://github.com/lightbend/akkaserverless-docs/pull/1193

**Review by patriknw — APPROVED** [review]
LGTM

**Pull request merged**